src/pages/login/loginFunctions.tsx

```diff
@@ -0,0 +1,23 @@
+export default interface userInput {
```

Comment:
this should be moved to type.ts file and should start with capital 
export interface UserInputProps {

}